src/Symfony/Component/String/Slugger/AsciiSlugger.php

```diff
@@ -103,6 +117,10 @@ public function slug(string $string, string $separator = '-', string $locale = n
             $transliterator = (array) $this->createTransliterator($locale);
         }
 
+        if ($this->enableEmoji) {
+            $transliterator[] = EmojiTransliterator::create($locale);
```

Comment:
I have a design question here

On this method (`slug`) the `$locale` could be `null`.
But the locale is mandatory in the `EmojiTransliterator`.

1. Should we check here that the locale is not null?
2. Should we disable the use the transl iterator if `$locale === null`
3. Should we keep the code as it

Anyway, some error could still occurs if someone pass an invalid locale

Reply:
IMHO 2, as create() doesnt accept null.

Also create() can return null, which AFAIK should be accounted for here.

Reply:
Done ✅